<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login UC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget #centralwidget {
	
	
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 212, 255), stop:1 rgba(113, 93, 255, 255));
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 127);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_error">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>Error</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="show_details">
            <property name="styleSheet">
             <string notr="true">QToolButton{
color: rgb(0, 0, 0);
font: bold 8pt &quot;Segoe UI&quot;;
}
QToolButton:hover{
color: rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Ver detalles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_popup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-radius: 8px;	
	background-image: url(:/Close_Image/graphics/cil-x.png);
	background-position: center;	
	background-color: rgb(60, 60, 60, 150);
}
QPushButton:hover {
	background-color: rgb(50, 50, 50);	
	color: rgb(200, 200, 200);
}
QPushButton:pressed {
	background-color: rgb(35, 35, 35);	
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/Graphics/graphics/cil-x.png</normaloff>:/Graphics/graphics/cil-x.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0" colspan="3">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="jalexcode">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>260</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>65</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="2">
        <widget class="QWidget" name="central_container" native="true">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QWidget" name="cardview" native="true">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget #cardview{
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
}
QPushButton{
	background-color: rgb(43, 110, 255);
	border-radius: 5px;
	font: 14pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
	padding: 5px;
}
QPushButton::hover{
	background-color: rgb(1, 51, 188);
}
QPushButton::pressed{
	background-color: rgb(1, 43, 120);
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>30</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item row="3" column="0" colspan="2">
              <widget class="QWidget" name="widget_2" native="true">
               <property name="styleSheet">
                <string notr="true">QLabel{
	font: bold 10pt &quot;Calibri&quot;;
	color: rgb(47, 47, 47);
}
QLineEdit{
	font: 14pt &quot;Calibri&quot;;
	color: rgb(47, 47, 47);
	border-bottom: 1px solid rgb(227, 227, 227);
}
QLineEdit::focus{
	border-bottom: 2px solid  rgb(43, 110, 255);
	background-color: rgb(225, 225, 225);
}</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QWidget" name="widget" native="true">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Usuario</string>
                     </property>
                     <property name="buddy">
                      <cstring>user_line_edit</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLineEdit" name="user_line_edit">
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="placeholderText">
                      <string>nombre.apellidos</string>
                     </property>
                     <property name="clearButtonEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Constraseña</string>
                     </property>
                     <property name="buddy">
                      <cstring>passw_line_edit</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLineEdit" name="passw_line_edit">
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                     <property name="placeholderText">
                      <string/>
                     </property>
                     <property name="clearButtonEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLabel" name="state_label">
               <property name="styleSheet">
                <string notr="true">color: rgb(121, 121, 121);
font: bold 12pt &quot;Calibri&quot;;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>210</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>210</width>
                 <height>100</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="app_resources.qrc">:/graphics/captiveportal-logo..png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QPushButton" name="login_button">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>LOGIN</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>120</width>
                 <height>335</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="wait_message" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget #wait_message{
	border-radius: 10px;
	background-color: rgba(0, 0, 0, 100);
}
QLabel{
	font: 16pt &quot;Calibri&quot;;
	color: rgb(255, 255, 255);
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>30</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Espere...</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>203</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>203</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>68</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin_UC">
    <property name="title">
     <string>Login UC</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="info_action">
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QMenu" name="menuEnlaces">
     <property name="title">
      <string>Enlaces</string>
     </property>
     <addaction name="github_action"/>
     <addaction name="gitlab_action"/>
     <addaction name="telegram_action"/>
     <addaction name="facebook_action"/>
    </widget>
    <addaction name="about_action"/>
    <addaction name="menuEnlaces"/>
   </widget>
   <addaction name="menuLogin_UC"/>
   <addaction name="info_action"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionGitLab">
   <property name="text">
    <string>GitLab</string>
   </property>
  </action>
  <action name="actionTelegram">
   <property name="text">
    <string>Telegram</string>
   </property>
  </action>
  <action name="github_action">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="gitlab_action">
   <property name="text">
    <string>GitLab</string>
   </property>
  </action>
  <action name="telegram_action">
   <property name="text">
    <string>Telegram</string>
   </property>
  </action>
  <action name="facebook_action">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="app_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
